frontend/src/views/auth/Login.vue

```diff
@@ -1,86 +1,97 @@
 <template>
-  <v-container class="fill-height justify-center align-center" fluid>
-    <v-col cols="12" sm="8" md="4">
-      <v-card class="elevation-12">
-        <v-toolbar color="green" dark elevation="1">
-          <v-toolbar-title>Login</v-toolbar-title>
-          <v-spacer />
-          <v-btn color="green darken-3" :to="{ name: 'register' }">Register</v-btn>
-        </v-toolbar>
-        <v-card-text>
-          <v-alert v-if="error" type="error">Login failed</v-alert>
-          <v-form @submit.prevent="login">
-            <v-text-field
-              id="inputUsername"
-              v-model="username"
-              label="Username"
-              name="username"
-              prepend-icon="mdi-account"
-              type="text" />
+  <v-container class="sso--container fill-height justify-center align-center" fluid>
+    <v-col cols="12" sm="8"
+           md="5" lg="4"
+           class="pa-0 pa-sm-4"
+           :class="{'fill-height': $vuetify.breakpoint.xsOnly}">
+      <v-card class="pa-8"
+              :tile="$vuetify.breakpoint.xsOnly"
+              :class="{'fill-height': $vuetify.breakpoint.xsOnly}">
+        <h1 class="display-1 text-center">Anmelden</h1>
```

Comment:
Können wir alles was oberhalb des `h1` steht in eine eigene Komponente packen? Dann musst `sso-container` auch nicht mehr global definieren.